dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/build.gradle

```diff
@@ -1,16 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2022 IBM Corporation and others.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License 2.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-2.0/
- * 
- * SPDX-License-Identifier: EPL-2.0
- *
- * Contributors:
- *     IBM Corporation - initial API and implementation
- *******************************************************************************/
-
-dependencies {
-  requiredLibs project(':io.openliberty.mpTelemetry.1.0.thirdparty')
-} 
```

Comment:
I think this is still needed. Although we are only calling the OTel API on the server (where it's provided by liberty), the test process will need the API classes to be available so that the classes are loadable when building the archives using ShrinkWrap.